pkg/output/output.go

```diff
@@ -111,3 +112,20 @@ func (o *Output) Summary(debug bool) {
 func (o *Output) Parse() ([]error, []error, []error) {
 	return o.failures, o.exceptions, o.errors
 }
+
+// GetEgressURLFailures returns only errors related to network egress failures.
+// Use the EgressURL() method to obtain the specific url for each error.
+func (o *Output) GetEgressURLFailures() []*handledErrors.GenericError {
```

Comment:
Something to consider - does this provide value or should we just return a list of strings represnting the egressURLs

Reply:
imo that makes sense. We can define better utils for output, to provide a smoother experience for callers
Also, it'd eliminate the need of parsing the output: https://gitlab.cee.redhat.com/service/uhc-clusters-service/-/merge_requests/5256/diffs#5e0a821c90e90a1732a77f0021d9754deffdb39b_474_474 (at some point, we need to bump-up the version here too 😅 )